<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2016, Christian Kvasny</author>
 <class>PrinterSettingEdit</class>
 <widget class="QDialog" name="PrinterSettingEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Druckereinstellungen Editieren</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Drucker Alias:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTableView" name="tableView"/>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="name">
            <property name="placeholderText">
             <string>Definitionname</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QrkPushButton" name="deleteButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>LÃ¶schen</string>
              </property>
              <property name="icon">
               <iconset resource="../../qrk.qrc">
                <normaloff>:/src/icons/eraser.png</normaloff>:/src/icons/eraser.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QrkPushButton" name="newButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="acceptDrops">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Neu</string>
              </property>
              <property name="icon">
               <iconset resource="../../qrk.qrc">
                <normaloff>:/src/icons/new.png</normaloff>:/src/icons/new.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Drucker:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QrkPushButton" name="okButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:src/icons/ok.png</normaloff>:src/icons/ok.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QrkPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>Abbrechen</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:src/icons/cancel.png</normaloff>:src/icons/cancel.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
 </tabstops>
 <resources>
  <include location="../../qrk.qrc"/>
  <include location="../../qrk.qrc"/>
 </resources>
 <connections/>
</ui>
